docs/konflux-onboarding.md

```diff
@@ -141,6 +141,9 @@ If you plan to use your catalogs on multiple cluster architectures, you will nee
 
 A sample PR for these changes can be found in https://github.com/konflux-ci/olm-operator-konflux-sample/pull/65.
 
+### Storing the FBC fragments built
```

Comment:
Consider rewriting this to something like `### Using Private repos for built FBC fragments`

Reply:
Updated